src/main/java/ui/components/pickers/MilestonePickerDialogResponse.java

```diff
@@ -0,0 +1,27 @@
+package ui.components.pickers;
+
+import javafx.scene.control.ButtonType;
+
+import java.util.Optional;
+
+/**
+ * This class is meant for the MilestonePickerDialog to pass the user's milestone selection back to the
+ * MilestonePicker for processing
```

Comment:
Write based on what the class represents rather than how it is used currently. It represents a milestone related button action on a UI. As you can see, it is hard to describe because it is not a very good abstraction. Anyway, we may have to live with it for now.